packages/reactive-element/src/decorators/query-assigned-nodes.ts

```diff
@@ -79,7 +79,8 @@ export function queryAssignedNodes(
  *
  * Note the type of this property should be annotated as `Array<Node>` if used
  * without a `selector` or `Array<HTMLElement>` if a selector is provided.
- * Please use `@queryAssignedElements` if using a CSS selector is desired.
+ * Please use {@linkcode queryAssignedElements @queryAssignedElements} if using
+ * a CSS selector is desired.
```

Comment:
This comment makes sense in the context of deprecating the arg to queryAssignedNodes and creating the new queryAssignedElements... But to a user coming in without this background, it doesn't make as much sense. 

That is, why use `queryAssignedElements`? Because I want to see only elements... and (maybe) filter them using a CSS selector.

```suggestion
 * Use {@linkcode queryAssignedElements @queryAssignedElements} to
 * list only elements, and optionally filter the element list using a CSS selector. 
 ```
 
Given that this only shows up in code suggestions for the deprecated form... maybe this suggestion is overkill. (Although I think some people are currently doing `queryAssignedNodes('', true, '*')` to get elements, in which case they don't really want the selector, they just want queryAssignedElements.)
 
If you don't take this suggestion, maybe just make the language active ("if using a CSS selector is desired" => "if you are using a CSS selector")

Reply:
I like this suggestion! It's clearer!